R/tm_g_waterfall.R

```diff
@@ -411,53 +399,53 @@ srv_g_waterfall <- function(id,
           show_value <- .(show_value)
         })
       )
-      teal.code::chunks_push_new_line()
+      q2 <- teal.code::eval_code(q1, "")
```

Comment:
In a few places we have this, I wonder if we can skip it by either having a blank line at the end of the previous quosure or the start of the next one

Reply:
we use bquote/substitute to make a calls - one can't add a new line to the quote as it will be ignored. If we want to add a line to the previous eval_code we would have to use `sprintf` instead of `substitute/bquote` and operate on the strings :/